.reuse/dep5

```diff
@@ -0,0 +1,24 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: alpaka
+Upstream-Contact: Jan Stephan <j.stephan@hzdr.de>
```

Comment:
This should probably someone else or a more general e-mail address.

Reply:
Uwe told me, that is not possible. We cannot create E-Mail addresses for organizations or projects.